tests/test_lock_db.py

```diff
@@ -47,82 +46,76 @@ def _get_file(self, *args):
 )
 
 
-os.environ['AUDB_CACHE_ROOT'] = pytest.CACHE_ROOT
-os.environ['AUDB_SHARED_CACHE_ROOT'] = pytest.SHARED_CACHE_ROOT
+def lock_paths(cache):
+    r"""Return list of lock file locations."""
+    paths = []
+    for version in DB_VERSIONS:
+        paths.append(
+            audeer.path(
+                cache,
+                DB_NAME,
+                version,
+                '.lock',
+            )
+        )
+        paths.append(
+            audeer.path(
+                cache,
+                DB_NAME,
+                version,
+                audb.Flavor().short_id,
+                '.lock',
+            )
+        )
+    return paths
 
 
 @pytest.fixture(
     scope='function',
     autouse=True,
 )
 def fixture_ensure_lock_file_deleted():
```

Comment:
The name of this and a few other fixtures still start with `fixture_*`. But for most fixture functions we don't stick to this naming convention. So I would also rename the remaining once.